src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -1188,13 +1189,20 @@ antlrcpp::Any CypherMainVisitor::visitCallProcedure(MemgraphCypher::CallProcedur
   const auto &maybe_found =
       procedure::FindProcedure(procedure::gModuleRegistry, call_proc->procedure_name_, utils::NewDeleteResource());
   if (!maybe_found) {
-    throw SemanticException("There is no procedure named '{}'.", call_proc->procedure_name_);
+    const auto mg_specific_name = procedure::gCallableAliasMapper.FindAlias(call_proc->procedure_name_);
```

Comment:
Isn't the Aliasing logic already implemented in the FindProcedure?

Reply:
It is used in two places. Here, it is used just specifically check for `mgps.validate` so we can instantiate the hardcoded cursor when creating the physical plan, the other is called from the traditional procedure cursor, to actually load a procedure from the module registry.